tests/functions.rs

```diff
@@ -179,6 +273,15 @@ compile_pass! {
         assert.deepEqual(api.to_vec([]), []);
         assert.deepEqual(api.to_vec([0n, 1n, 2n, 3n, 4n]), [0n, 1n, 2n, 3n, 4n]);
     ),
+    (
+    export class Api {
+        constructor();
+
+        static fetch(url, imports): Promise<void>;
```

Comment:
what does static mean in this context?

Reply:
It's a static function, so you can use it without a class instance. But this is actually not static here now ..